pyproject.toml

```diff
@@ -133,3 +133,14 @@ ignore = [
   "src/tests/*",
   "SECURITY.md",
 ]
+
+[tool.poetry]
+name = "pretalx"
+version = "2024.2.0.dev0"
+description = "Conference organisation: CfPs, scheduling, much more"
+authors = ["Eventyay <info@eventyay.com>"]
+
+[tool.poetry.dependencies]
```

Comment:
Same as eventyay-tickets repo. This dependency list already exists in the same file. Consider using PDM instead of Poetry when working with a _pyproject.toml_ file like this.

Reply:
Hi @hongquan, fixed